### ISSUE TITLE
Optimal Control of Building Energy Systems

### ISSUE THREAD

**Issue by Perference2025** [pmohebi2/Battery-Sizing#2]
Hello, Parastoo. Submitting this question might be the only way to reach you. I was fortunate to read your article titled "Comparing different parameter identification techniques for optimal control of building energy systems". The question I have is that when I was doing RC model identification, the fitting was good in the training set; but once it came to the validation set, the error significantly increased, and the coupling parameters "drifted" to an unreasonable range, resulting in poor performance of subsequent MPC/PI control (it was difficult to balance energy consumption and temperature).
What strategy would you recommend to make the parameters both identifiable and friendly to control? For example: adding physical priors/regularization in the centralized identification? The most common root cause of identification failure is insufficient coupling excitation or inconsistency between the objective function and the control task? 
If possible, could you please communicate via email? Thank you for taking the time to consider my request. I look forward to your response.
